index.html

```diff
@@ -123,7 +129,7 @@ <h3 class="work-name">Multi-Post Stories</h3>
                         <li class="lang">ruby</li>
                     </ul>
                 </div>
-                <button class="see-project">See Project</button>
```

Comment:
Pop-up is missing buttons on the medium view. Moreover, please follow the font defined in the design.  Kindly see the screenshot below and fix the issues.
![tablet](https://user-images.githubusercontent.com/35504149/132556333-19277fee-5fbf-47a6-96c1-b8e3d22d52bb.png)